src/Components/FlashBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Banner, Sans } from "@artsy/palette"
+
+interface Props {
+  messageCode: string
+}
+
+const textMessage: (msg: string) => React.FC = (message: string) => () => (
+  <Sans color="white100" size="3" lineHeight={1} weight="medium">
+    {message}
+  </Sans>
+)
+
+const messages: { [k: string]: React.FC } = {
+  confirmed: textMessage("Your email has been confirmed."),
+  // TODO: I don't think we should show this as a fallback as any string could trigger it
+  fallback: textMessage("An error has occurred. Please try again."),
+}
+
+export const FlashBanner: React.FunctionComponent<Props> = ({
+  messageCode,
+}) => {
+  const Message = messages[messageCode] || messages.fallback
+  return (
+    <Banner dismissable backgroundColor="black100">
+      <Message />
```

Comment:
This looks tricker than it should be. I think it's more straight forward to inline the `<Sans color="white100" size="3" lineHeight={1} weight="medium">`:

```tsx
const MESSAGES = {
   confirmed: "Your email has been confirmed.",
  // other keys...
 }
```

```tsx
<Banner dismissable backgroundColor="black100">
  <Sans color="white100" size="3" lineHeight={1} weight="medium">
    {MESSAGES[messageCode] || "An error has occurred. Please try again."}
  </Sans>
</Banner>
```

Reply:
So there are two reasons I did this, though maybe there is a better way.
1. the text size on the default palette banner does not match the mockups, so i'm adding a second `Sans` wrapper to lower it. maybe not ideal.
2. The designs for the 'email confirmation has been sent. `[send again]`' have a button, which suggests that the children might be more dynamic than just text (though I guess we could put a button inside the tag above).

Also as an aside I think we are going to drop the fallback message for unhandled flashMessage keys.